src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -102,4 +121,97 @@ public CompletableFuture<Void> handlePutRequest(String endpoint, JsonObject reco
 
     return future;
   }
+
+  public  CompletableFuture<JsonObject> post(String endpoint, JsonObject recordData) {
+    CompletableFuture<JsonObject> future = new FolioVertxCompletableFuture<>(ctx);
+
+    if (logger.isDebugEnabled()) {
```

Comment:
As option we can delete this checking to increase conditional coverage

Reply:
Since log4j switched to "DEBUG" level in testing mode - the code inside "if" block covered by tests. I think it worth to leave it as is